cmd/podman/manifest/inspect.go

```diff
@@ -38,6 +38,8 @@ func init() {
 	_ = inspectCmd.RegisterFlagCompletionFunc(authfileFlagName, completion.AutocompleteDefault)
 	flags.BoolP("verbose", "v", false, "Added for Docker compatibility")
 	_ = flags.MarkHidden("verbose")
+	flags.StringVar(&inspectOptions.Format, "format", "", "inspect manifest list with provided format")
```

Comment:
As is, I’m rather unhappy with adding an option
- where the actual semantics (“raw”) differs from the claimed one (“OCI”)
- with such a generic name, with a fair risk of someone external using it (`--internal-unstable-output-raw-format` would work better)
- which _silently_ does nothing for remote clients

Would be possible to run `skopeo inspect --raw` instead of adding this partially-implemented option (and instead of taking the effort to implement it fully)?

At least _some_ parts of `test/system` seem to depend on the Skopeo binary already.

Reply:
Done using skopeo now.